src/pages/create-page/index.tsx

```diff
@@ -3,29 +3,36 @@ import { Input } from "../../ui/input";
 import { Textarea } from "../../ui/textarea";
 import "./styles.css";
 import { Button } from "../../ui/button";
+import { useNotes } from "../../modules/hooks/use-notes";
+import { useState } from "react";
 
 const CreatePage = () => {
-  // const { createNote } = useNotes();
+  const { createNote } = useNotes();
+  
+  const [titulo, setTitulo] = useState('')
+  const [descricao, setDescricao] = useState('')
 
-  //   try {
-  //     const response = await createNote();
-  //   } catch (e) {}
-  // };
+  const handleSubmit = async () => {
+    const note = {titulo, descricao}
+      try {
+        const response = await createNote(note);
```

Comment:
Em `const response = await createNote(note);`, se o _response_ for o esperado, pode usar um _console_ para imprimir uma mensagem de sucesso para o usuário (console temporário porque ainda não criamos o _toast_).

Porém, se o _response_ não for o esperado, use o `throw`

```
if (!response)
    throw new Error("Invalid response when trying to create note");
```